### PR TITLE
Add installation instructions to readme.

### PR DESCRIPTION
**Pull request by idvorkin**
Adding installation guidance #48

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -27,3 +27,15 @@ Requirements
 ============
 
 These completions requires clink@0.4.3 or higher version
+
+Installation
+============
+
+Clone git into your clink settings directory
```
Comment (vladimir-kotikov) [inline README.md]: One should not clone git, but git repo :)

**Comment by vladimir-kotikov** [comment]
@idvorkin, it seems that these commands could be replaced by one: `git clone https://github.com/vladimir-kotikov/clink-completions %LOCALAPPDATA%/clink` (see SO question [here](http://stackoverflow.com/a/651079/4113568) )

Also i can't find in clink documentation if it is really searches for lua files in this location. Could you point me to that?

**Comment by idvorkin** [comment]
_Excellent questions, this is based on my quick testing, so holler if I'm confused ;) _

The reason you can't call git clone is because you can only clone into an empty directory.  Clink files will already be in the directory.    Also, the reason you don't want to call git init in that directory every time is because if you have local changes they will be destroyed.

**Comment by idvorkin** [comment]
RE: Clink documentation  on file location -- that's confusing.   (How do I notify @mridgers about this question)? From: https://github.com/mridgers/clink/blob/master/docs/clink.md:

> Clink looks for Lua scripts in the folders as described in the Configuring Clink section. By default Ctrl-Q is mapped to reload all Lua scripts which can be useful when developing and iterating on your own scripts.

And

> Settings and history are persisted to disk from session to session. The location of these files depends on which distribution of Clink was used. If you installed Clink using the .exe installer then Clink uses the current user's non-roaming application data directory. This user directory is usually found in one of the following locations;

       c:\Documents and Settings\<username>\Local Settings\Application Data (XP)
       c:\Users\<username>\AppData\Local (Vista onwards)
       The .zip distribution of Clink creates and uses a directory called profile which is located in the same directory where Clink's core files are found.

> All of the above locations can be overridden using the --profile <path> command line option which is specified when injecting Clink into cmd.exe using clink inject.

**Comment by vladimir-kotikov** [comment]
> How do I notify `@mridgers` about this question

You just did it :smile:

So, cloning this to `%localappdata%` is not universal. I'd just provide a short overview of how it is possible to determine athe actual Clink's profile location.

**Comment by lijunle** [comment]
I think just mention that clone into the setting folder and the default setting folder is `%LOCALAPPDATA%` folder is enough. No need to provide the commands. If the user ever customize the clink settings location, s/he knows the meaning of *setting folder*.

**Comment by lijunle** [comment]
BTW, you don't need to the exact setting folder. The subfolders under setting folder are fine.

**Comment by idvorkin** [comment]
Closing as it's now stale.

**Pull request closed without merging**